integration/apply_test.go

```diff
@@ -35,7 +35,7 @@ func TestDiagnoseRenderApply(t *testing.T) {
 		MarkIntegrationTest(t.T, NeedsGcp)
 		ns, client := SetupNamespace(t.T)
 
-		out := skaffold.Diagnose("--yaml-only").InDir("examples/multi-config-microservices").RunOrFailOutput(t.T)
+		out := skaffold.Diagnose("--yaml-only").InDir("testdata/multi-config-pods").RunOrFailOutput(t.T)
```

Comment:
nit:  in the PR description you mention:
```
refactor tests builds cross-platform/multi-platform examples/microservice project to use example/multi-pods, as cross-platform build is very slow when running, go build ..., we should make project source code as simple as possible when using cross-platform/multi-platform builds.
```
example/multi-config-microservices doesn't actually use `cross-platform/multi-platform` builds, it only use multiple skaffold configs/modules.  I'm not sure this needs to be changed, was this actually slowing things down?  This doesn't actually use cross-platform/multi-platform builds

Reply:
@aaron-prindle it should be alright since the multi-platform values are being provided on line 45 using the flag `--platform=linux/amd64,linux/arm64`

Reply:
@aaron-prindle We're running integration tests against 3 types of cluster, and our CI machine is amd architecture, so when testing against arm cluster, the machine has to build cross-platform image with binfmt interpreter, this can be 10X slow compared to building image with native platform. Reducing the project source code complexity helps a little bit in build speed. You can test this by running `` skaffold build -p gcb --platform="linux/arm64"  --default-repo=${}``, in skaffold/examples/microservice folder  it usually takes 10 minutes to finish. 

Then replace everything in leery-app/app.go , leery-web/app.go with very simple code like 
```go
package main

import (
	"fmt"
)

func main() {
	for {
		fmt.Println("...")
	}
}
```
should be able to reduce the building time to 3 mins, still very slow : (

Reply:
+1 to the original logic/changes, sorry for the misunderstanding and thanks for the explanation!